tnz/zti.py

```diff
@@ -1838,6 +1846,45 @@ def help_plugin(entry=entry):
 
         self.plugins = " ".join(plugins)
 
+    def __register_macros(self):
+        import importlib.util
+        import sys
+        import types
+
+        if not os.path.isdir(self.macros_dir):
```

Comment:
The name of the default macro dir is very zti specific. I would have a hard time believing the name would be used for something else. So, some error messages might be in order if things don't look right. Could help the user if they  do things wrong accidentally.